internal/ocm/resource/cluster.go

```diff
@@ -71,16 +75,22 @@ func (c *Cluster) CreateNodes(autoScalingEnabled bool, replicas *int64, minRepli
 		if minReplicas != nil {
 			minReplicasVal = int(*minReplicas)
 		}
-		if err := validations.MinReplicasValidator(minReplicasVal, multiAZ, false, 0); err != nil {
-			return err
+		// TODO need to identify the private subnet or remove this validation from TF
+		if clusterTopology == rosa.Classic {
+			if err := validations.MinReplicasValidator(minReplicasVal, multiAZ, clusterTopology == rosa.Hcp, 0); err != nil {
+				return err
+			}
 		}
```

Comment:
The condition `clusterTopology == rosa.Hcp` will always be false because we enter the `if` statement only if `clusterTopology == rosa.Classic`. Is this intended?

Reply:
Yes, this is intended as the validation for HCP is different. I'm currently leaving the validation for the API which I find more guaranteed anyways

Reply:
ok, but then I believe it would be more readable to pass directly `false`:
```suggestion
		if clusterTopology == rosa.Classic {
			if err := validations.MinReplicasValidator(minReplicasVal, multiAZ, false, 0); err != nil {
				return err
			}
		}
```
WDYT?